units/src/amount.rs

```diff
@@ -514,7 +524,7 @@ fn is_too_precise(s: &str, precision: usize) -> Option<usize> {
 
 const INPUT_STRING_LEN_LIMIT: usize = 50;
 
-/// Parse decimal string in the given denomination into a satoshi value and a
+/// Parses a decimal string in the given denomination into a satoshi value and a
 /// bool indicator for a negative amount.
```

Comment:
I don't know if this is what we are doing now, but since `bool` is a Rust type we can backtick it right?

```suggestion
/// `bool` indicator for a negative amount.
```

Reply:
maybe doclink as well?

Reply:
Done